src/aktualizr_info/main.cc

```diff
@@ -79,7 +79,9 @@ int main(int argc, char **argv) {
     ("images-target",  "Outputs targets.json from Image repo")
     ("images-targets",  "Outputs targets.json from Image repo")
     ("image-target",  "Outputs targets.json from Image repo")
-    ("director-target",  "Outputs targets.json from Director repo");
+    ("director-target",  "Outputs targets.json from Director repo")
+  // Identify the database is for secondary ECU
+    ("secondary",  "the database is for secondary ECU");
```

Comment:
I added a parameter for aktualizr-info, it means if you want to make the log for secondary database looks better, we need to add this parameter to command line when the database is for secondary. Does it make sense for all of our scenario ?